phoenix-core/src/it/java/org/apache/phoenix/end2end/IndexRebuildTaskIT.java

```diff
@@ -129,45 +128,42 @@ public void testIndexRebuildTask() throws Throwable {
             Task.addTask(conn.unwrap(PhoenixConnection.class), PTable.TaskType.INDEX_REBUILD,
```

Comment:
@gokceni Above, we have upserted data into the view. Won't this automatically trigger an index build i.e. won't global view index get these populated rows already? Why do we need to explicitly run IndexRebuildTask?

Reply:
removed